db/migrate/20180315080842_add_subscriber_id_to_emails.rb

```diff
@@ -0,0 +1,5 @@
+class AddSubscriberIdToEmails < ActiveRecord::Migration[5.1]
+  def change
+    add_reference :emails, :subscriber
```

Comment:
in the reverted commit we've also added a `foreign_key` (https://github.com/alphagov/email-alert-api/pull/527/files#diff-5ada1ec3c318e9d3712d888134d92ec5R4) - any reason we aren't doing that here?

Reply:
Oh yeah, I should clarify this difference in the PR. The reason is that adding the foreign key locks the tables for a few minutes so we can't deploy this easily during working hours.

Reply:
Cool. I wonder if using `NOT VALID` on the creation of the foreign key would help speed that up?

Reply:
I hadn't seen `NOT VALID` before! It looks like that would help, but it seems like there's no support for it in active record so we would need to use an `execute` statement in the migration.

Reply:
I've pushed up a commit which creates the foreign key using `NOT VALID`.

Reply:
awesome - I was thinking it might be a bit messy to include in this pr, but 👍